client/me/purchases/payment-methods/components/payment-method-edit-form-fields.tsx

```diff
@@ -0,0 +1,86 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import FormFieldAnnotation from 'calypso/my-sites/checkout/composite-checkout/components/form-field-annotation';
+import {
+	LeftColumn,
+	RightColumn,
+} from 'calypso/my-sites/checkout/composite-checkout/components/ie-fallback';
+import useCountryList from 'calypso/my-sites/checkout/composite-checkout/hooks/use-country-list';
+import TaxCountrySelectMenu from './tax-country-select-menu';
+
+const GridRow = styled.div`
+	display: -ms-grid;
+	display: grid;
+	width: 100%;
+	-ms-grid-columns: 48% 4% 48%;
+	grid-template-columns: 48% 48%;
+	grid-column-gap: 4%;
+	justify-items: stretch;
+`;
+
+const FieldRow = styled( GridRow )`
+	margin-top: 16px;
+
+	:first-of-type {
+		margin-top: 0;
+	}
+`;
+
+const RenderEditFormFields = ( {
```

Comment:
You might want to consider trying the existing `TaxFields` component in place of `RenderEditFormFields`, which looks very similar to this and might save some effort.

https://github.com/Automattic/wp-calypso/blob/f6c4d023458daccfc70b8b2fac647da71859c4f6/client/my-sites/checkout/composite-checkout/components/tax-fields.tsx#L32-L46

Reply:
I dug into this, and from what I found, it looks like it may require some rewriting of how things currently work. Rather than prolonging this diff further, I decided to go with the suggestion of just removing the `renderEditForm` and sticking the form code directly into the variable from your other suggestion.

> `const taxForm = ( <form><div className="contact-fields payment-methods__tax-fields"> ... </form )`